frontend/heart-code/src/components/start-page/StartPage.css

```diff
@@ -12,7 +12,7 @@
 }
 
 .login-register-button {
-  width: 20%;
+  width: 86px !important;
```

Comment:
hier kun je de width beter laten voor wat het is, bij de registerButton valt bij mij de tekst er een beetje uit, ze resizen automatisch als je width weglaat